### PR TITLE
added erf function to theano and tf backends

### PR DESCRIPTION
**Pull request by small-yellow-duck**
Added erf function to theano and tf backends.

### CODE REVIEW

File: keras/backend/theano_backend.py [hunk 1]
```diff
@@ -661,6 +661,10 @@ def cos(x):
     return T.cos(x)
 
 
+def erf(x):
```
Comment (fchollet) [inline keras/backend/theano_backend.py]: Add a docstring.

**Comment by Dref360** [comment]
Is there a common use-case for this?
Also please add a test.

File: keras/backend/tensorflow_backend.py [hunk 2]
```diff
@@ -1593,6 +1593,18 @@ def cos(x):
         A tensor.
     """
     return tf.cos(x)
+    
+    
+def erf(x):
+    """Computes erf of x element-wise.
```
Comment (ahundt) [inline keras/backend/tensorflow_backend.py]: Please include a non abbreviated explanation of what erf is.

**Comment by fchollet** [comment]
Closing seemingly abandoned PR

**Pull request closed without merging**